src/mailroom.py

```diff
@@ -0,0 +1,99 @@
+"""Code to create email and report from a list of donors."""
+donors = []
+
+
+class Donor:
+    """Stores donor info."""
+
+    first_name = ""
+    last_name = ""
+    donations = 0
+    total_amount = 0
+
+    @classmethod
+    def create(cls, first_name, last_name, donations, total_amount):
+        """Create an instance of Donor for testing."""
+        d = Donor()
+        d.first_name = first_name
+        d.last_name = last_name
+        d.donations = donations
+        d.total_amount = total_amount
+        return d
+
+
+def initial_prompt():
```

Comment:
There is a difference in this function between the instructions and the correct option (Create vs create, etc...)